ecosystem/platform/server/app/views/settings/profile.html.erb

```diff
@@ -36,24 +36,26 @@
 <% delete_account_dialog = DialogComponent.new(id: 'delete_account_dialog')
    verif_number = Random.rand(10_000..100_000) %>
 
-<div class="mt-16">
+<div class="mt-10">
   <div>
-    <%= render ButtonComponent.new(size: :small, scheme: :secondary, class: 'border-white-100 p-4', dialog: delete_account_dialog) do %>
+  <div class="flex justify-end">
+    <%= render ButtonComponent.new(size: :small, class: 'bg-transparent text-red-400 uppercase font-normal px-0 py-0', dialog: delete_account_dialog) do %>
       delete account
     <% end %>
+    </div>
```

Comment:
I don't think this is supposed to be here, but if it is necessary, can you fix the indentation (and make sure it's balanced with opening tags)?